install_config/topics/ocp_osp_provisioning.adoc

```diff
@@ -189,8 +189,19 @@ openshift_node_open_ports:
 # set in the inventory file as openshift_openstack_kuryr_pod_subnet_cidr.
 # By default, this value is /24.
 
-# The value of openshift_portal_net in OSEv3.yml must also match the following value:
-# openshift_openstack_kuryr_service_subnet_cidr: *"172.23.192.0/18"*
+# The value of openshift_portal_net in OSEv3.yml must be within the range given
+# for openshift_openstack_kuryr_service_subnet_cidr (usually half of it),
+# ensuring it does not overlap with the range provided by
+# [openshift_openstack_kuryr_service_pool_start, openshift_openstack_kuryr_service_pool_end].
+# openshift_portal_net is the range used by services and Octavia Load Balancers
+# VIPs, while amphora VMs get Neutron ports on the range specified by
+# [openshift_openstack_kuryr_service_pool_start, openshift_openstack_kuryr_service_pool_end]:
+# [OSEv4.yml]
```

Comment:
This is 4 rather than 3, right? Just confirming.

Reply:
ups... no no, that is a typo, this is OSEv3.yml